### PR TITLE
fix(shared-data): ot3 trash below deck

### PR DESCRIPTION
**Pull request by sfoster1**
The bottom of the OT-3 fixed trash is actually below the deck. Without
this change, attempts to move to the top of the "well" of the fixed
trash will cause out-of-bounds z moves, particularly during cancel.

Closes RCORE-122

### CODE REVIEW

File: shared-data/labware/definitions/2/opentrons_1_trash_3200ml_fixed/1.json [hunk 1]
```diff
@@ -45,6 +45,6 @@
   "cornerOffsetFromSlot": {
     "x": 0,
     "y": 0,
-    "z": 0
+    "z": -94
```
Comment (mcous) [inline shared-data/labware/definitions/2/opentrons_1_trash_3200ml_fixed/1.json:48]: The overal `dimensions.zDimension` of this labware def in `93.5`. Does it need to be updated to `94`, does this number need be `-93.5`, or does `wells.A1.z` need to be `-0.5`?

File: shared-data/labware/definitions/2/opentrons_1_trash_3200ml_fixed/1.json [hunk 2]
```diff
@@ -30,7 +30,7 @@
       "depth": 187.5,
```
Comment (mcous) [inline shared-data/labware/definitions/2/opentrons_1_trash_3200ml_fixed/1.json:30]: This wasn't touched by this PR, but this `depth` looks wrong

**Review by mcous — DISMISSED** [review]
See inline comments

File: shared-data/labware/definitions/2/opentrons_1_trash_3200ml_fixed/1.json (hunk 2 above)
Comment (sfoster1) [reply shared-data/labware/definitions/2/opentrons_1_trash_3200ml_fixed/1.json:30]: nope, it's correct. On the OT-3, the built-in trash extends both below and above the level of the deck, to give it more space for tips. It really is 187.5mm deep, with a bottom 94mm below deck level and a top 93.5mm above deck level.

File: shared-data/labware/definitions/2/opentrons_1_trash_3200ml_fixed/1.json (hunk 1 above)
Comment (sfoster1) [reply shared-data/labware/definitions/2/opentrons_1_trash_3200ml_fixed/1.json:48]: I think the absolute correct thing for it to be would be 187.5, which would correctly represent the overall size of the labware's z bounding box from bottom to top. But, half of that is below the deck... so +93.5 is accurate for the z bounding box above the deck level

**Comment by sfoster1** [comment]
> See inline comments

Honestly not quite sure what the right way to do this is; this labware system wasn't really made with the concept of stuff being below the deck in it. The current definitions do make the machine move to the physically correct place, though.

**Review by mcous — APPROVED** [review]
Approving to unblock testing, as discussed in Slack. Deck def and motion planning system will require more work to adapt to negative deck locations

**Pull request merged**